CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,94 @@
+# Release notes for 1.13.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since 1.12.0
+
+### Feature
+
+- Adds the flag --extra-modify-metadata, which when set to true, will inject extra PVC and PV metadata as parameters when calling ModifyVolume on CSI Drivers (#420, @mdzraf)
```

Comment:
Can you change to this format?
([#420](https://github.com/kubernetes-csi/external-snapshotter/pull/420), [@mdzraf](https://github.com/mdzraf))

Reply:
done.